### PR TITLE
[v1.1] Model adapter CI/CD add domain adapter

### PR DESCRIPTION
**Pull request by zhouyu5**
- add ci/cd scripts for doamin adapter
- modify run scripts for unet

### CODE REVIEW

File: modelzoo/unet/scripts/run_dist.sh [hunk 1]
```diff
@@ -15,31 +15,36 @@
 #!/bin/bash
 set -x
 
+echo "############################## setting env ##############################"
+master="vsr214"
+hosts="vsr214,vsr215,vsr216,vsr218"
```
Comment (XinyaoWa) [inline modelzoo/unet/scripts/run_dist.sh:20]: Hostname should not appeared, please use args to input.

Comment (zhouyu5) [reply modelzoo/unet/scripts/run_dist.sh]: fixed

**Comment by zhouyu5** [comment]
@xuechendi please have a code review~

**Review by XinyaoWa — APPROVED** [review]

**Review by xuechendi — APPROVED** [review]
approved

**Pull request merged**